config.js

```diff
@@ -7,35 +7,35 @@ function getConfig(env) {
     case 'mainnet':
         return {
             networkId: 'mainnet',
-            nodeUrl: 'https://rpc.mainnet.nearprotocol.com',
+            nodeUrl: 'https://rpc.mainnet.near.org',
             contractName: CONTRACT_NAME,
-            walletUrl: 'https://wallet.mainnet.nearprotocol.com',
-            helperUrl: 'https://helper.mainnet.nearprotocol.com',
+            walletUrl: 'https://wallet.mainnet.near.org',
+            helperUrl: 'https://helper.mainnet.near.org',
         };
     case 'development':
     case 'testnet':
         return {
             networkId: 'default',
-            nodeUrl: 'https://rpc.testnet.nearprotocol.com',
+            nodeUrl: 'https://rpc.testnet.near.org',
             contractName: CONTRACT_NAME,
-            walletUrl: 'https://wallet.testnet.nearprotocol.com',
-            helperUrl: 'https://helper.testnet.nearprotocol.com',
+            walletUrl: 'https://wallet.testnet.near.org',
+            helperUrl: 'https://helper.testnet.near.org',
         };
     case 'devnet':
         return {
             networkId: 'devnet',
-            nodeUrl: 'https://rpc.devnet.nearprotocol.com',
+            nodeUrl: 'https://rpc.devnet.near.org',
             contractName: CONTRACT_NAME,
-            walletUrl: 'https://wallet.devnet.nearprotocol.com',
-            helperUrl: 'https://helper.devnet.nearprotocol.com',
+            walletUrl: 'https://wallet.devnet.near.org',
+            helperUrl: 'https://helper.devnet.near.org',
```

Comment:
`https://helper.devnet.near.org/` not working

Reply:
cert error

Reply:
@chadoh I'm working with Render to get these up

Reply:
@chadoh  Render is configured. Please, try again

Reply:
Yes, looks like `devnet` works. `betanet` and `testnet` are still giving cert errors, though